lib/nostrum/struct/event/message_delete_bulk.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Nostrum.Struct.Event.MessageDeleteBulk do
+  @moduledoc """
+  Struct representing a Message Delete Bulk event
+  """
+
+  alias Nostrum.Snowflake
+
+  defstruct [
+    :channel_id,
+    :guild_id,
+    :ids
+  ]
+
+  @typedoc "Channel id of the deleted message"
+  @type channel_id :: Snowflake.t()
+
+  @typedoc """
+  Guild id of the deleted message
+
+  `nil` if a non-guild message was deleted.
+  """
+  @type guild_id :: Snowflake.t() | nil
+
+  @typedoc "Ids of the deleted messages"
+  @type ids :: [Snowflake.t(), ...]
```

Comment:
I think the `...` might be a little confusing here, it almost seems like we're implying there can be `any` type items in here, and not just snowflakes.

Reply:
No confusion from my perspective. According to the official docs, `[Snowflake.t(), ...]` is syntactical sugar for `nonempty_list(Snowflake.t())`. The delete message bulk event should contain at least one message id, so this typespec looks appropriate.

Reply:
Yeah, you're completely correct.